Dockerfile

```diff
@@ -44,6 +44,7 @@ RUN ./kapitan/inputs/helm/build.sh \
 FROM base AS runtime
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
+      curl \
```

Comment:
That's gonna improve the debugging experience when running Commodore in Docker quite a bit :+1: